Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:5.5.0
+
+MAINTAINER Go About <tech@goabout.com>
+
+RUN apt-get install fontconfig
```

Comment:
This will probably not work. (See [the Dockerfile best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#apt-get).) Common is something like:

```
RUN apt-get update -q && \
    DEBIAN_FRONTEND=noninteractive apt-get install -yq \
        PACKAGE1 \
        PACKAGE2 \
    && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```